rethinkdb_base_block.py

```diff
@@ -0,0 +1,94 @@
+import rethinkdb as rdb
+
+from nio.block.base import Block
+from nio.command import command
+from nio.properties import (IntProperty, StringProperty, VersionProperty,
+                            TimeDeltaProperty)
+from nio.util.discovery import not_discoverable
+
+
+@command("Get server info", method="_get_server_info")
+@command("Reconnect to server", method="_reconnect_connection")
+@command("List server databases", method="_get_list_of_databases")
+@command("List database tables", method="_get_list_of_tables")
```

Comment:
I was thinking it would be useful to have some easy to get information in the builder as a way of, say, checking what tables exist before choosing which one to watch for changes.